server/src/models/FaqModel.js

```diff
@@ -25,6 +25,7 @@ const FaqSchema = new Schema(
     },
   },
   { strict: true },
+  { timestamps: true },
```

Comment:
```suggestion
  { strict: true, timestamps: true },
```
For the options for a schema, they need to all be in the same object